src/Microsoft.ML.Data/Microsoft.ML.Data.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="System.Threading.Channels" Version="4.7.1" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" Version="$(SystemThreadingTasksDataflowPackageVersion)" />
```

Comment:
We capture our repo wide dependency versions in build\Dependencies.props. Can you please add it there and use a named version like you see in the lines above and below?

Reply:
Updated. Thanks for catching this!